extensions/test-src/org/pentaho/platform/web/http/api/resources/UserRoleDaoResourceTest.java

```diff
@@ -109,6 +107,12 @@ public void testAssignRoleToUser() {
     String user = "testUser1";
     String roles = "testRole1";
 
+    userRoleResource = spy( userRoleResource );
+    IPentahoSession session = mock( IPentahoSession.class );
```

Comment:
Isn't it possible to use `StandaloneSession` here? I'd prefer to mock as less, as possible